docs/ruleset_functions/get_zone_peak_internal_load.md

```diff
@@ -0,0 +1,57 @@
+# get_zone_peak_internal_load
+**Schema Version:** 0.0.21
+**Description:** finds the peak coincident internal loads of a zone and returns the value in W/sf
+
+**Inputs:** 
+- **RMR**
+- **zone**
+
+**Returns:**  
+- **result**: an array giving 5 values: [total peak watts in the zone, total zone area, full load hours] the total peak watts is the internal coincident peak loads in all spaces in the zone - full load hours is the total of all full load hours for the space, normalized by the individual load.
+- For example: if a space has 1 W/sf lighting (2000 hours/year), 0.5 W/sf misc equipment (8000 hours/year), and no occupancy, the full_load_hours would be:
+	- (1 / 1.5) * 2000 + (0.5 / 1.5) * 8000 = 4000
```

Comment:
Please rename the function so that it's clear that it returns both peak internal gains, EFLH, and area. 

The values should be IP, not metric, because RCT convert values to IP once it parces the RMD.

I am not quite following how effecitve full load hours are calculated, and not sure what they mean in the context of this function. I thikn that in order to determine peak internal load of the zone we need to do the following:
a. determine the hourly loads from occupants, lighting and equipment for each space within the zone. E.g., for misc. equipment you'd calcualte power x mutiplier_schedule to get hourly loads. Or did we agree to assume schedule fraction of 1 to capture peak loads? 
b. add up occupant, lighting and equipment load for each hour to get the hourly total internal load for the spae. (Need to do unit conversion to account for difference in units between occupants and lighting+equipment.)  
c. add up the values for each spaces in the zone to get hourly loads for the zone.
d. get the maximum value

Reply:
regarding a change to IP units, the schema denotes for interior lighting that the units are W/m2 ([https://github.com/open229/ruleset-model-description-schema/blob/master/docs229/ASHRAE229.schema.md#InteriorLighting]), so I do the unit conversion from W/m2 to btu/hr/ft2?

for effective full load hours, I was trying to capture the number of hours that the zone is at the peak load.  From my own modeling, I sometimes use a high W/sf with a low schedule value when I want to capture a space that very rarely has high internal gains.  I am sure we will change the function, but I was trying to give us a place to start the discussion.